_data/people/richa.yml

```diff
@@ -0,0 +1,12 @@
+active: true
+focus-area:
+  - Particle Physics
+  - Machine Learning
+institution: University of Puerto Rico, Mayaguez
+past_institution: { institution: Panjab University Chandigarh end_date: 2022-12-19 }
```

Comment:
I think you're missing a `,` here, but also I didn't find the `past_institution` key anywhere

Reply:
Sorry about that. I saw it in somebody's yml file. So assumed it might exist. But it's not needed anyway for me because I wasn't a part of iris-hep then. So removed it.